understanding/identify-purpose.html

```diff
@@ -34,47 +34,47 @@ <h1>達成基準 1.3.6: 目的の特定を理解する</h1>
       <main>
          <section id="intent">
             <h2>意図</h2>
-            <p>The intent of this success criterion is to support personalization and preferences in order for more people to use the web, communicate, and interact with society.</p>
-            <p>Familiar terms and symbols are key for users with a limited vocabulary to being able to use the web. However, what is familiar for some users may not be for other users so programmatically associating user-interface components and icons enables people to load a set of symbols that is appropriate for them.</p>
-            <p>This success criteria requires the author to add the context, propose, and meaning of symbols, regions, buttons, links, and fields so that user agents knows what they do and can adapt them to make them understandable for the user. It is achieved by adding semantics or metadata that provides this context. It is similar to adding role information (as required by 4.2.1) but instead of providing information about what the UI component is (such as an image) it provides information about what the component represents (such as a link to the home page).</p>
-            <p>Identifying regions of the page allows people to remove or highlight regions with their user agent.</p>
-            <p>Products for people who are non-vocal often use symbols to help users communicate. These symbols are in fact people&#39;s language. Unfortunately, many of these symbols are both subject to copyright and are not interoperable. That means end-users can only use one device, and cannot use content, apps, or assistive technology that has not been made by a single company.</p>
-            <p>This success criteria enables symbols to be interoperable so that symbol users can understand different content that was not just made by one company. When users&#39; symbols are mapped to the same nodes then user agents can load the user-understandable symbol. That means people could buy the symbols and use them across different devices or applications. (Note that the symbols would still be proprietary, but they could then be interoperable.)</p>
+            <p>この達成基準の意図は、より多くの人々がウェブを使用し、コミュニケーションを取り、そして社会と交わるために、パーソナライズ及び好みをサポートすることである。</p>
+            <p>使い慣れた用語や記号は、語彙が限られている利用者にとってウェブを使えるようにするための手がかりだ。しかしながら、一部の利用者にとって馴染みのあるものは他の利用者にとっては馴染みのないものなので、ユーザーインタフェース コンポーネントとアイコンをプログラムによって関連づけることで、利用者は自分に適した一連の記号を読み込むことができる。</p>
+            <p>この達成基準は、制作者がシンボル・領域・ボタン・リンク及びフィールドのコンテキスト・提案及び意味を追加して、ユーザエージェントが動作を認識し、利用者が理解できるようにすることを要求する。それは、この文脈を提供するセマンティック又はメタデータを追加することで達成される。role情報を追加するのと似ているが (4.2.1で要求されているように) 、UIコンポーネントが何であるかについての情報 (たとえば画像) を提供する代わりに、それはコンポーネントが何を表すものかについての情報 (たとえばホームページのリンクなど) を提供する。</p>
```

Comment:
１文目の

> This success criteria requires the author to add the context, propose, and meaning of symbols, regions, buttons, links, and fields so that user agents knows what they do and can adapt them to make them understandable for the user.

ですが、並列するものが多くて非常に理解しづらい文章に・・:cry:合っているかも自信が。。

Reply:
雑なコメントですが、本来的には
add (A, B, and C) to (D, E and F)
になるのが正しい感じなのでしょうか…？（andが2回出現することからして）